package.json

```diff
@@ -6697,6 +6697,379 @@
+                                                "name": "+Refactor",
+                                                "icon": "edit",
+                                                "type": "bindings",
+                                                "bindings": [
+                                                    {
+                                                        "key": "a",
+                                                        "name": "Execute code actions",
+                                                        "icon": "lightbulb",
+                                                        "type": "command",
+                                                        "command": "editor.action.codeAction"
+                                                    },
+                                                    {
+                                                        "key": "o",
+                                                        "name": "Organize imports",
+                                                        "icon": "edit",
+                                                        "type": "command",
+                                                        "command": "editor.action.organizeImports"
+                                                    },
+                                                    {
+                                                        "key": "r",
+                                                        "name": "Rename symbol",
+                                                        "icon": "edit",
+                                                        "type": "command",
+                                                        "command": "editor.action.rename"
+                                                    },
+                                                    {
+                                                        "key": "R",
+                                                        "name": "Refactor actions",
+                                                        "icon": "edit",
+                                                        "type": "command",
+                                                        "command": "editor.action.refactor"
+                                                    },
+                                                    {
+                                                        "key": "e",
+                                                        "name": "Extract to function or variable",
+                                                        "icon": "gather",
+                                                        "type": "command",
+                                                        "command": "editor.action.codeAction",
+                                                        "args": {
+                                                            "kind": "refactor.extract"
+                                                        }
```

Comment:
can you move this up, so that the file is alphabetically ordered?

Reply:
Done